index.html

```diff
@@ -0,0 +1,42 @@
+<html>
+<head>
+  <title>Tic tac toe</title>
+  <link rel="stylesheet" href="style.css" media="screen" type="text/css" />
+
+</head>
+	<body onload="startGame()";>
```

Comment:
Got it Matt. Thanx for your feedback. 


     On Tuesday, July 7, 2015 9:43 AM, Matt Scilipoti <notifications@github.com> wrote:
   

 In index.html:> @@ -0,0 +1,42 @@
> +<html>
> +<head>
> +  <title>Tic tac toe</title>
> +  <link rel="stylesheet" href="style.css" media="screen" type="text/css" />
> +
> +</head>
> +	<body onload="startGame()";>
All javascript should live in your javascript file. You can use the AddEventListener you learned about this week.—
Reply to this email directly or view it on GitHub.